testbench/common.py

```diff
@@ -814,10 +823,22 @@ def handle_retry_test_instruction(database, request, socket_closer, method):
                 "Injected 'connection reset by peer' fault", 500
             )
         elif items[0] == "broken-stream":
+            print("broken stream")
             return __get_streamer_response_fn(database, method, socket_closer, test_id)
     broken_stream_after_bytes = (
         testbench.common.retry_return_broken_stream_after_bytes.match(next_instruction)
     )
+
+    retry_stall_beginning_matches = testbench.common.retry_stall_beginning.match(
+        next_instruction
+    )
+    if retry_stall_beginning_matches and method == "storage.objects.get":
+        media = request.args.get("alt", None)
+        if media == None or media != "json":
+            print("handle_retry_test: retry_stall_beginning matches")
+            database.dequeue_next_instruction(test_id, method)
+            time.sleep(10)
```

Comment:
You'll want to create a derivative of the following helper function:
https://github.com/raj-prince/storage-testbench/blob/0655bd3d968c7189ac37bbde56bde776b19c1a27/testbench/common.py#L695

This function will break connection but you can make it stall after certain number of bytes are returned to the client.